### DISCUSSION TITLE
can't see weekends!

### DISCUSSION THREAD

**Discussion by uheist** [FossifyOrg/General-Discussion#417] [Ideas]
![photo_2024-01-17_20-20-51](https://github.com/FossifyOrg/Calendar/assets/156951823/9e9dde95-939d-4320-abbc-2ab494eeea6b)

Cant see the weekenddates and daynames  (7 day view selected!)
Same behavior like in simplemobiletools

Thank you

**Comment by Aga-C** [comment]
It's already raised here: https://github.com/FossifyOrg/Calendar/issues/41. You can add your screenshot there as another theme-related bug.
